skywalking-collector/skywalking-collector-cluster/src/main/java/com/a/eye/skywalking/collector/actor/AbstractClusterWorker.java

```diff
@@ -11,18 +11,45 @@
 import org.apache.logging.log4j.Logger;
 
 /**
+ * The <code>AbstractClusterWorker</code> should be implemented by any class whose instances
+ * are intended to provide receive remote message that message will transfer across different jvm
+ * which running in same server or another address server.
+ * <p>
+ * Usually the implemented class used to receive persistence data or aggregation the metric.
+ *
  * @author pengys5
+ * @since feature3.0
  */
 public abstract class AbstractClusterWorker extends AbstractWorker {
 
+    /**
+     * Constructs a <code>AbstractClusterWorker</code> with the worker role and context.
+     *
+     * @param role           The responsibility of worker in cluster, more than one workers can have
+     *                       same responsibility which use to provide load balancing ability.
```

Comment:
If multi-workers are for load balance, they should be more likely called worker instance. Meaning, each worker have multi instances.